e2e-tests/cypress/tests/integration/channels/emoji/custom_emoji_2_spec.ts

```diff
@@ -120,7 +120,9 @@ describe('Custom emojis', () => {
 
         cy.get('#emojiPicker').should('be.visible').within(() => {
             // # Scroll to start of custom category section
-            cy.findByLabelText('emoji_picker.custom').should('exist').click().wait(TIMEOUTS.FIVE_SEC);
+            cy.get('i.icon-emoticon-custom-outline').as('customSection');
+            cy.get('@customSection').click().wait(TIMEOUTS.FIVE_SEC);
+            cy.get('@customSection').click().wait(TIMEOUTS.FIVE_SEC);
```

Comment:
Just curiosity. Why do we need two clicks?

Reply:
Yes, the emoji section re-renders when we click on custom section since the whole emoji pack is loading